Library/ViewModels/DiscoveryFiltersViewModel.swift

```diff
@@ -281,7 +281,9 @@ private func topFilters(forUser user: User?) -> [DiscoveryParams] {
 
   if user != nil {
     filters.append(.defaults |> DiscoveryParams.lens.starred .~ true)
-    filters.append(.defaults |> DiscoveryParams.lens.recommended .~ true)
+    filters.append(.defaults 
+                   |> DiscoveryParams.lens.recommended .~ true 
+                   <> DiscoveryParams.lens.backed .~ false)
```

Comment:
ah cool! I forgot where to find these ops and didn't know how to use 'em, but this makes sense 🙇